src/main/java/io/vertx/ext/httpservicefactory/package-info.java

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
We should also document the URL format that is assumed, and how to use it for bintray etc.